src/pages/registration/contributors_tab/components/AddContributorModalContent.tsx

```diff
@@ -1,71 +1,90 @@
 
   return (
     <>
       {initialSearchTerm && (
-        <StyledSubHeading variant="h6">
-          {t('registration:contributors.prefilled_name')}: {initialSearchTerm}
-        </StyledSubHeading>
+        <Typography variant="subtitle1">
+          {t('registration:contributors.prefilled_name')}: "{initialSearchTerm}"
+        </Typography>
       )}
-      <SearchBar handleSearch={(newSearchTerm) => setSearchTerm(newSearchTerm)} initialSearchTerm={initialSearchTerm} />
+      <StyledTextField
+        variant="outlined"
+        fullWidth
+        value={searchTerm}
+        onChange={(event) => setSearchTerm(event.target.value)}
+        autoFocus
+        placeholder={t('common:search_placeholder')}
+        label={t('common:search')}
+        inputProps={{ 'data-testid': 'search-input' }}
+        InputProps={{
+          startAdornment: <SearchIcon />,
+        }}
+      />
+
       {isLoadingAuthorities ? (
         <StyledProgressWrapper>
           <CircularProgress size={100} />
         </StyledProgressWrapper>
-      ) : authorities && authorities.length > 0 ? (
-        <>
-          {searchTerm && (
-            <AuthorityList
-              authorities={authorities}
-              selectedArpId={selectedAuthor?.id}
-              onSelectAuthority={setSelectedAuthor}
-              searchTerm={searchTerm}
-            />
-          )}
-          <StyledButtonContainer>
-            <Button
-              color="secondary"
-              startIcon={<AddIcon />}
-              data-testid="connect-author-button"
-              disabled={!selectedAuthor}
-              onClick={() => selectedAuthor && addAuthor(selectedAuthor)}
-              size="large"
-              variant="contained">
-              {t('common:add')}
-            </Button>
-          </StyledButtonContainer>
-        </>
+      ) : authorities && authorities.length > 0 && debouncedSearchTerm ? (
+        <AuthorityList
+          authorities={authorities}
+          selectedArpId={selectedAuthor?.id}
+          onSelectAuthority={setSelectedAuthor}
+          searchTerm={debouncedSearchTerm}
+        />
       ) : (
-        searchTerm && <Typography>{t('common:no_hits')}</Typography>
+        debouncedSearchTerm && <Typography>{t('common:no_hits')}</Typography>
       )}
+
+      <DialogActions>
+        <Button onClick={handleCloseModal}>{t('common:close')}</Button>
+        <Button color="primary" data-testid="button-create-new-author" onClick={openNewAuthorModal}>
+          {t('contributors.create_new_author')}
+        </Button>
+        <Button
+          color="secondary"
+          startIcon={<AddIcon />}
+          data-testid="connect-author-button"
+          disabled={!selectedAuthor}
+          onClick={() => selectedAuthor && addAuthor(selectedAuthor)}
+          size="large"
+          variant="contained">
+          {t('common:add')}
+        </Button>
+      </DialogActions>
```

Comment:
Vi har gjort det litt knotete for oss selv sånn som vi har lagd `Modal` komponenten vår. Tror kanskje det hadde vært enklere å flytte mest mulig av props og ting vi setter i `Modal` over til theme, så kunne vi brukt alt som det skal direkte:
```
<Dialog ...>
   <DialogTitle ...>
   <DialogContent ...>
   <DialogActions ...>
</Dialog>
```
I dag er det ofte knotete å  ha bare en `DialogActions`, uten at den blir inni `DialogContent` eller noe sånn. Gidder ikke å ta det akkurat nå, og denne PRen har noen sånne "feil" :/

Reply:
kanskje skrive det opp en plass uansett da? enten på en samleoppgave på Jira eller på Confluence?